docs/content/guides/auth/azureAD.md

```diff
@@ -0,0 +1,41 @@
+# Authentication Guide: Azure Active Directory
+
+1. Login or Sign-up on portal.azure.com</br>
+
+2. Navigate to Azure Active Directory from the homepage or the sidebar.</br>
+![azure-active-directory-navigation](../../images/auth/azure-active-directory-navigation.png)</br>
+
+3. Navigate to App Registration</br>
+![azure-active-directory-new-registration](../../images/auth/azure-active-directory-new-registration.png)</br>
+
+4. Enter Name "HedgeDoc". Then click on Register</br>
+![azure-active-direcotry-register](../../images/auth/azure-active-directory-register.png)</br>
+
+5. In the next page, click on `Add a certificate or secret` then navigate to `Client Secrets`.</br>
+Create a new client secret, add a description of your choice and copy the secret value.</br>
+![azure-active-directory-new-client-secret](../../images/auth/auzre-active-directory-new-secret.png)</br>
+
+
+6. Navigate to authentication page, then add a new platform. Select web as the platform.</br>
+![azure-active-directory-authentication](../../images/auth/azure-active-directory-authentication.png)</br>
+
+7. Set the `Redirect URI` to `https://YOURHOSTNAME/auth/oauth2/callback`. Check `ID Tokens` and uncheck `Access Token`.</br>
+![azure-active-directory-redirect-uri](../../images/auth/azure-active-directory-redirect-uri.png)</br>
+
+8. Retrieve the APPLICATION-ID and DIRECTORY-ID from the "Overview" section.</br>
+![azure-active-directory-overview](../../images/auth/azure-active-directory-overview.png)</br>
```

Comment:
Please provide proper texts for the images.

Reply:
Should I create a new commit for the changes or should I amend my previous commit?

Reply:
Amend your previous commit.

Reply:
I have made the required changes. Please let me know if any other changes are required.